.env.example

```
@@ -130,8 +130,82 @@ GENERATE_MARKDOWN=true
+
+# [OPTIONAL] Enable rotated text filtering (default: true)
+FILTER_ROTATED_TEXT=true
+
+# [OPTIONAL] Rotation method (default: bbox_orientation)
+# Options: "bbox_orientation" (uses element bbox coordinates)
+ROTATION_METHOD=bbox_orientation
+
+# [OPTIONAL] Rotation angle thresholds in degrees (default: 25.0-65.0)
+# Text rotated between these angles is filtered (diagonal watermarks)
+ROTATION_MIN_ANGLE=25.0
+ROTATION_MAX_ANGLE=65.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ROTATION_MAX_ANGLE key should go before the ROTATION_MIN_ANGLE key

(UnorderedKey)